test/testsuite/lantest.c

```diff
@@ -43,8 +59,26 @@ char    *Path;
 int     Version = 34;
 int     Mac = 0;
 char    *Test = NULL;
+static uint16_t vol;
+static DSI *dsi;
+static char    *Server = "localhost";
+static int     Port = DSI_AFPOVERTCP_PORT;
+static char    *Password = "";
+static int     Iterations = 2;
+static int     Iterations_save;
+static int     iteration_counter = 0;
+static struct timeval tv_start;
+static struct timeval tv_end;
+static struct timeval tv_dif;
+static pthread_t tid;
+static int active_thread = 0;  /* Track if we have an active thread */
+
+/* Global Debug flag */
+int Debug = 0;
+/* Global Quiet flag override for lantest (stdout output enabled) */
+int lantest_Quiet = 0;
```

Comment:
Why do we need an override for the global Quiet flag? All the other tools use it and I don't think there's any conflict.

Reply:
So this was the source of my previous pain with stdout suppression..
Simply, I needed Quiet to be accessible by the new `lantest_io_monitor` module. However if I exposed `Quiet` from lantest.c, it conflicted with the already exposed Quiet in afpcli.c (https://github.com/Netatalk/netatalk/blob/main/test/testsuite/afpcli.c#L10) - which is currently defaulted to 1 and the root cause of the stdout suppression.

This is why non-error fprintf lines have been using stderr everywhere, which is not nice.
It was either update afpcli.c to default to 0 (ideal), or override it with a sidecar name to keep it local to afp_lantest for now. Both Quiet and Debug should be 0 by default IMO.

Reply:
The thing is in the spectest, we have two distinct layers of verbose output. The first level (verbose), Quiet = 0, logs AFP method calls in addition to some other informative messages. The second level (very verbose), Debug = 1, logs DSI responses, in addition to everything from the first level.

I'm open to rethinking how to structure this. The most important requirement is to keep the DSI logging and AFP logging separate, in my opinion, since they're both so chatty but yet important for troubleshooting test failures.

Details to worked out in a future PR. :)

Reply:
Yea, We already have -v and -V for this.
So
```
-b = Show everything
-V = Very verbose
-v = verbse
(Nothing) = Normal
-q = Very little
```

This is the standard? its debatable whether -b comes before or after -Vv in this hierarchy, but I don't see an issue to make this better

Reply:
Looks good to me. But perhaps we don't need something above -V very verbose, if we introduce the -q Extra quiet mode?